assets/app/view/game/round/stock.rb

```diff
@@ -318,7 +318,8 @@ def render_bank_companies
             },
           }
 
-          @game.buyable_bank_owned_companies.map do |company|
+          companies = @step.respond_to?(:bank_owned_companies) ? @step.bank_owned_companies : @game.buyable_bank_owned_companies
```

Comment:
why not just override this in your game? respond_to? in general is not a great way to go

Reply:
The step in this case has the knowledge of which company is being formed, but the game doesn't know that.  I'm not sure how to get the game to know about it without game doing the same thing (@step.respond_to?).  I was following the pattern that shows up several other times in stock.rb.

Reply:
the game knows what step it is though

Reply:
Changed to override in game.